doc-site/docs/architecture/distributed_sequencer.md

```diff
@@ -0,0 +1,1437 @@
+ 
+### Coordinator switchover
+When the block height reaches a range boundary then all nodes independently reach a universal conclusion the new coordinator.
+
+Lets consider the case where the scenario explored above continues to the point where the blockchain moves to block 4.  In the interim, while node A was coordinator, nodes B, C and D delegated transactions `B-1`, `B-2`, `B-3`,  `C-1`, `C-2`, `C-3`, `D-1`, `D-2` and `D-3`,  and A itself has assembled `A-1`, `A-2`, `A-3`.
+
+`A-1` , `B-1`, `C-1` and `D-1` have already been submitted to base ledger and confirmed.  
+`A-2` , `B-2`, `C-2` and `D-2` have been prepared for submission, assigned a signing address and a nonce but have not yet been mined into a confirmed block.  NOTE it is irrelevant whether these transactions have been the subject of an `eth_sendTransaction` yet or not.  i.e. whether they only exist in the memory of the paladin node's transaction manager or whether they exist in the mempool of node A's blockchain node.  What is important is that
+
+ - they have been persisted to node A's database as being "dispatched" and
+ - are not included in block 3 or earlier
+
+![image](../images/distributed_sequencer_switchover_frame1.svg){ width="500" }
+
+As all nodes in the group ( including node A) receive receipts of block 3, they will recognize node B as the new coordinator
+
+
+![image](../images/distributed_sequencer_switchover_frame2.svg){ width="500" }
+
+Node A will continue to process and monitor receipts for transactions `A-2` , `B-2`, `C-2` and `D-2` but will abandon transactions `A-3` , `B-3`, `C-3` and `D-3`.
+
+![image](../images/distributed_sequencer_switchover_frame3.svg){ width="500" }
+
+Node A sends a `DelegationReturned` message to the respective sender nodes for each non dispatched transaction that it had in flight.
+
+
+![image](../images/distributed_sequencer_switchover_frame4.svg){ width="500" }
+
+The final transmission from node A, as coordinator, is a message to node B with details of the speculative domain context predicting confirmation of transactions `A-2` , `B-2`, `C-2` and `D-2` .  This includes the state IDs for any new states created by those transactions and any states that are spent by those transaction.  This message also includes the transaction hash for the final transaction that is submitted on the sequence coordinated by node A.  Node B needs to receive this message so that it can proceed with maximum efficiency. Therefore this message is sent with an assured delivery quality of service (similar to state distribution).  
+
+Meanwhile, all nodes delegate transactions  `A-3` , `B-3`, `C-3` and `D-3` to node B.  
```

Comment:
I'm struggling to understand the assumptions for this "meanwhile".

What makes it safe to send anything to B, if you are not absolutely certain that A has transferred your previous transactions to B?

The message from C->B that is sent by C because `DelegationReturned ` was received by C could (in fact in the above sequence is reasonably likely) to reach B, before the message from A->B that provides information about B's previous transactions.

I wonder, given these facts:
- `DelegationReturned` will be sent by A->C after any message (sorry couldn't see the name) for `I've taken ownership of submission` for `C-2` is sent from A->C
- The sender is responsible for their transactions
- The sender thus will know at the point it sends C-3 to B, that C-2 has reached the point of confirmed delegation on A

... whether C should just package up the information about which confirmed delegations it has on A, up to B.